modules/ml/src/rtrees.cpp

```diff
@@ -422,6 +422,18 @@ Ptr<RTrees> RTrees::create()
     return makePtr<RTreesImpl>();
 }
 
+//Function needed for Python and Java wrappers
+Ptr<RTrees> RTrees::load(const String& filepath)
+{
+    FileStorage fs;
+    fs.open(filepath, FileStorage::READ);
+
+    Ptr<RTrees> rtree = makePtr<RTreesImpl>();
+
+    ((RTreesImpl*)rtree.get())->read(fs.getFirstTopLevelNode());
```

Comment:
Should this be done for the ANN and SVM too? I noticed that they still use this implimentation.